### PR TITLE
replace docker run for running etcdctl command with just etcdctl

### PR DESCRIPTION
**Pull request by reetasingh**
fix https://github.com/kubernetes/website/issues/36533
Problem:
[Set up a High Availability etcd Cluster with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) mentions that you can check cluster health by running a docker run <etcd-image> etcdctl command.

 Kubernetes doesn't support Docker Engine natively any more.

Solution:
For checking etcd cluster health we can just use etcdctl command directly instead of docker run

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 1]
```diff
@@ -273,22 +273,21 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
...
-   --key /etc/kubernetes/pki/etcd/peer.key \
-   --cacert /etc/kubernetes/pki/etcd/ca.crt \
-   --endpoints https://${HOST0}:2379 endpoint health --cluster
-   ...
-   https://[HOST0 IP]:2379 is healthy: successfully committed proposal: took = 16.283339ms
-   https://[HOST1 IP]:2379 is healthy: successfully committed proposal: took = 19.44402ms
-   https://[HOST2 IP]:2379 is healthy: successfully committed proposal: took = 35.926451ms
-   ```
+    ```sh
+    ETCDCTL_API=3 etcdctl \
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:277]:
```suggestion
    # If etcdctl isn't available, you can run this tool inside a container image.
    # You would do that directly with your container runtime using a tool such as
    # crictl run
    # - and not through Kubernetes.
    ETCDCTL_API=3 etcdctl \
```

Let's explain to people where `etcdctl` comes from.

Comment (reetasingh) [reply content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:281]: added the note

**Comment by tengqm** [comment]
/lgtm
again

File: content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 2]
```diff
@@ -273,22 +273,25 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
...
+    --key /etc/kubernetes/pki/etcd/peer.key \
+    --cacert /etc/kubernetes/pki/etcd/ca.crt \
+    --endpoints https://${HOST0}:2379 endpoint health
+    ...
+    https://[HOST0 IP]:2379 is healthy: successfully committed proposal: took = 16.283339ms
+    https://[HOST1 IP]:2379 is healthy: successfully committed proposal: took = 19.44402ms
+    https://[HOST2 IP]:2379 is healthy: successfully committed proposal: took = 35.926451ms
+    ```
+    
+    - Set `${ETCD_TAG}` to the version tag of your etcd image. For example `3.4.3-0`. To see the etcd image and tag that kubeadm uses execute `kubeadm config images list --kubernetes-version ${K8S_VERSION}`, where `${K8S_VERSION}` is for example `v1.17.0`.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:292]:
```suggestion
```

**Review by kundan2707 — COMMENTED** [review]
lgtm

**Comment by reetasingh** [comment]
Hi @sftim can you please review this PR?

**Pull request merged**